repos/fdbt-site/src/pages/selectCapValidity.tsx

```diff
@@ -8,7 +8,7 @@ import { getSessionAttribute } from '../utils/sessions';
 import { getAndValidateNoc, getCsrfToken, getErrorsByIds } from '../utils';
 import RadioConditionalInput from '../components/RadioConditionalInput';
 import { isCapExpiry } from '../interfaces/typeGuards';
-import { getFareDayEnd } from '../data/auroradb';
+import { getCapExpiry, getFareDayEnd } from '../data/auroradb';
```

Comment:
Can we add a back button to this page? So that the user can press back if they don't want to create one. We should also clear the errors from the session on the /viewCaps page.

Reply:
i don't think the back will make sense, as will be showing them the selected option and they can continue if they want. Let me know, do we still need back button.